src/interactableComponents/drawer/newDrawer.js

```diff
@@ -234,6 +238,7 @@ class NewDrawer extends PureBaseComponent {
         leftActionsContainerStyle={this.leftActionsContainerStyle}
         onSwipeableWillOpen={this.onSwipeableWillOpen}
         onSwipeableWillClose={this.onSwipeableWillClose}
+        useNativeAnimations={useNativeAnimations}
```

Comment:
you don't need to pass it, it will be passed in 'others' if you don't extract it from the props object.

Reply:
Done